homeassistant/components/fritzbox/diagnostics.py

```diff
@@ -0,0 +1,33 @@
+"""Diagnostics support for AVM Fritz!Smarthome."""
+from __future__ import annotations
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_COORDINATOR, DOMAIN
+from .coordinator import FritzboxDataUpdateCoordinator
+
+TO_REDACT = [CONF_USERNAME, CONF_PASSWORD]
```

Comment:
Sets offer faster lookup
```suggestion
TO_REDACT = {CONF_USERNAME, CONF_PASSWORD}
```

Reply:
ups 🙈  thx for fixing it 🙂